config/metadata/metadata.yaml

```diff
@@ -48,6 +48,10 @@ spec:
               value: aibrix-redis-master
             - name: REDIS_PORT
               value: "6379" 
+            # TODO: cache is shared across all control plane so add feature flags to enable metric pull
+            # for now setting to a 1 hr for metadata service
+            - name: AIBRIX_POD_METRIC_REFRESH_INTERVAL_MS
```

Comment:
does metadata service rely on any cache data?

Reply:
Just see below implementations. Actually, it still leverage cache information. not send request to every pods. In this case, we need to change model frequency to seconds or minute level. at least make sure new deployment metadata can be exposed shortly. 1hr is too long

Reply:
I mentioned in the previous reply that querying pods need api-key information which gateway does not have it. Till we figure out how to achieve that, using internal-cache to return models registered.

<img width="960" alt="image" src="https://github.com/user-attachments/assets/a5fef35a-e471-493c-9ca2-560dbb1661f4" />


Frequecy referenced here is not model refresh. Internal cache code is shared across all components and this frequecy is for metrics refresh (check the env variable name). I have added a todo to make cache's background workers such as refresh metrics or request trace to me have enable/disable flags.

Reply:
Agree. I tend to zone out and that's just for metrics not related to model. It looks good to me